consensus/polybft/signer/serialization.go

```diff
@@ -0,0 +1,70 @@
+package bls
+
+import (
+	"fmt"
+	"math/big"
+
+	bnsnark1 "github.com/0xPolygon/bnsnark1/core"
+)
+
+func BytesToBigInt2(bytes []byte) ([2]*big.Int, error) {
+	if len(bytes) != 64 {
+		return [2]*big.Int{}, fmt.Errorf("expect length 64 but got %d", len(bytes))
+	}
+
+	return [2]*big.Int{
+		new(big.Int).SetBytes(reverse(bytes[:32])),
+		new(big.Int).SetBytes(reverse(bytes[32:])),
+	}, nil
+}
+
+func BytesToBigInt4(bytes []byte) ([4]*big.Int, error) {
```

Comment:
Do these functions need to be public?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1082/commits/9d94ae1b96a471650e94a9732f1c44245a7cdc57